ohsomeTools/gui/ohsome_spec.py

```diff
@@ -449,7 +449,6 @@ def _get_selected_point_layers_geometries(self) -> {}:
         for item in point_layers_list:
             file_name = item.name()
             radius = self.params["RADIUS"]
-            ordered_layer_radii.append(int(radius))
```

Comment:
When not appending to `ordered_layer_radii` here, can we get rid of the variable?